tree/ntupleutil/v7/inc/ROOT/RNTupleExporter.hxx

```diff
@@ -39,6 +49,12 @@ public:
       std::string fOutputPath;
       std::uint64_t fFlags;
 
+      /// Optional filter that determines which columns are included or excluded from being exported.
+      /// By default, export all columns. If you only want to include certain column types, add them
+      /// to this set and change fFilterType to kWhitelist.
```

Comment:
I am guesstimating that this is *not* the common cases.  Rather than filtering on the data type, I expect the user to be want to filter/select in allmost all cases on the column names.

Reply:
Well, it is the only case I found so far (as the only user so far :) and for the ntupleutils I think our strategy is "implement things as they are requested". 
However I do think I should change the name to a more specific one (`fColumnTypeFilter`) to avoid name clashing in the future.

Reply:
~~Trimming (and skimming) data files is an essential and extremely common operation in our field.  Despite the improvement in file size for `RNTuple`, I don't see changing any time soon and I assume users would assume the feature will will soon be present.~~

Nevermind, I was confused.  This code change is in the `Exporter` (which is an internal research/exploration tools for `RNTuple` and where this behavior actually makes more sense) instead of the `RNTupleMerger`